### ISSUE TITLE
torch.normal_ with slicing

### ISSUE THREAD

**Issue by ailzhang** [pytorch/xla#885]
```
import torch
import torch_xla

device = 'xla:0'
# device='cpu'
q = torch.empty(100, 100, device=device)
q_row1 = q[0:1].clone()
q[99:100].normal_()
print(q[99:100])
print(q[99:100].std())
```
Expected behavior: q[99:100] should be normalized

**Comment by ailzhang** [comment]
I think this is along fixed along other issues, closing..
